src/components/Layout/items.js

```diff
@@ -0,0 +1,384 @@
+            >
+              International Human Rights Law
+            </a>,
+            'top3a'
+          ),
+          getItem(
+            <a href="/topics/religious-freedom" title="Religious Freedom">
+              Religious Freedom
+            </a>,
+            'top3b'
+          ),
+          getItem(
+            <a href="/topics/targeted-killing" title="Targeted Killing">
+              Targeted Killing
+            </a>,
+            'top3c'
+          ),
+          getItem(
+            <a
+              href="/topics/immigration-detention"
+              title="Immigration Detention"
+            >
+              Immigration Detention
+            </a>,
+            'top3d'
+          ),
+          getItem(
+            <a href="/topics/refugee-protection" title="Refugee Protection">
+              Refugee Protection
+            </a>,
+            'top3e'
+          ),
+        ]
+      ),
+      getItem(
+        <a href="/campaigns/innovation-lab-tech-for-human-rights" title="">
+          The Use & Abuse of Technology
+        </a>,
+        'top4',
+        [
+          getItem(
+            <a href="/resource/counter-human-rights-disinformation" title="">
+              Disinformation
+            </a>,
+            'top4a'
+          ),
+          getItem(
+            <a
+              href="/topics/human-rights-defenders"
+              title="Human Rights Defenders"
+            >
+              Human Rights Defenders
+            </a>,
+            'top4b'
+          ),
+          getItem(
+            <a href="/resource/hold-human-rights-abusers-accountable" title="">
+              Accountability
+            </a>,
+            'top4c'
+          ),
+        ]
+      ),
+    ]
+  ),
+  getItem(
+    <a href="/resources" title="Resources">
+      Resources
+    </a>,
+    'res'
+  ),
+  getItem(
+    <a href="/press" title="">
+      Media
+    </a>,
+    'med',
+    [
+      getItem(
+        <a href="/press" title="">
+          Press Releases
+        </a>,
+        'med1'
+      ),
+      getItem(
+        <a href="/about/events" title="">
+          Events
+        </a>,
+        'med2'
+      ),
+      getItem(
+        <a href="/blog" title="Blog">
+          Blog
+        </a>,
+        'med3'
+      ),
+      getItem(
+        <a href="/press/in-the-news" title="">
+          In the News
+        </a>,
+        'med4'
+      ),
+    ]
+  ),
+  getItem(<span title="">Initiatives</span>, 'init', [
+    getItem(
+      <a href="/campaigns/innovation-lab-tech-for-human-rights" title="">
+        Innovation Lab
+      </a>,
+      'init1'
+    ),
+    getItem(
+      <a href="https://www.vfai.org" title="">
+        Vets for American Ideals
+      </a>,
+      'init2'
+    ),
+  ]),
+  getItem(<span title="Asylum Case Data">Asylum Case Data</span>, 'data', [
+    getItem(
+      <a href="/table" title="table">
+        Table
+      </a>,
+      'data1'
+    ),
+  ]),
+];
```

Comment:
Consider in the future creating a function that takes the distinct values within each of these getItem() calls and iterates through the set to affect this same structure. It would keep the code compact and enable visualizing the relationship between the disparate nested layers.